src/core/lib/iomgr/resolve_address.h

```diff
@@ -66,4 +66,11 @@ extern grpc_error *(*grpc_blocking_resolve_address)(
     const char *name, const char *default_port,
     grpc_resolved_addresses **addresses);
 
+/* Returns GRPC_ERROR_CANCELLED by default. If it's overriden and returns
```

Comment:
Some tests re-implemented grpc_blocking_resolve_address, but the async resolver does not use  this method. grpc_customized_resolve_address is provided as a unified interface for the tests to implement their own address resolving solutions.